Cargo.toml

```diff
@@ -25,6 +25,7 @@ bytes = "1"
 mime = "0.3.14"
 sha1 = "0.10"
 httpdate = "1"
+language-tags = "0.3.2"
```

Comment:
I've tried scrolling through to find the answer, but failed so far. We don't expose this dependency, right?

Reply:
We are, actually. `LinkValue::href_lang` returns `Option<&[LanguageTag]>`.

Reply:
So you want that to change before merging?

Reply:
I'd rather not expose this dependency publicly, at least that's my default feeling.

Reply:
Personally I feel like it shouldn't be such a big deal for `headers`, as opposed to `headers-core`. But I'll look into how complex it would be to remove it from public API / what the consequences would be exactly.

Reply:
So the main reason I would like to not change the code here is that it allows `push_href_lang` to be infallible. If `language-tags` was required to be removed from public API, that would mean parsing the language tag provided by the user in that function. Please let me know whether you agree or not and I'll update the PR accordingly.